docs/Meadow/Meadow.OS/Cellular/index.md

```diff
@@ -19,31 +31,54 @@ Then, to configure your network you just need to follow these three steps:
 ```yaml
 Settings:
     APN: virtueyes.com.br # Access Point Name
+    Module: BG95M3 # Module model (BG770A, BG95M3 or M95)
     User: virtu # APN user (optional)
     Password: virtu # APN password (optional)
     Operator: 72410 # Carrier numeric operator code (optional)
-    Mode: 7 # Network mode (Cat-M1 = 7, Nb-IoT = 9)
+    Mode: CATM1 # Network mode (CATM1, NBIOT or GSM) (optional)
 ```
 
-***Notes**: It's recommended to define the carrier operator code, since it avoids the module network automatic selection, making the connection generally more reliable.*
+> **Notes**: If the carrier numeric operator code (**Operator**) or the network mode is not specified (**Mode**), the module will attempt to automatically determine the optimal network based on the M2M sim card inserted and your location. However, if you encounter any connectivity issues, it is advisable to manually configure these settings.
 
-2 - Select **BG770A** as **DefaultInterface** in the **meadow.config.yaml**, if you don't have this *yaml* file in your device, you should create and flash it to the device:
+2 - Select `Cell` as `DefaultInterface` in the **meadow.config.yaml**, if you don't have this *yaml* file in your device, you should create and flash it to the device:
 
 ```yaml
 Network:
-    DefaultInterface: BG770A
+    DefaultInterface: Cell
 ```
 
-3 - Reserve the pins **I9**, **H13** and **C7**, which are used to turn on the **BG770A Cell Wing** and to exchange data with the **F7 Feather v2**:
+3 - Reserve the pins `I9` and `H13`, which are used to exchange data between the modem and the **Meadow F7v2 Feather**, and the `C7`, which is used to turn on the modules:
 
 ```yaml
 Device:
     ReservedPins: I9;H13;C7
 ```
 
+> **Notes**: You can find the pins `I9`, `H13` and `C7` as `D00`, `D01` and `D10` on the **Meadow F7v2 Feather**, respectively.
+
 ## Hardware configuration
```

Comment:
Should I put some images here? I can try to take some showing the setup @alexischr

Reply:
We can add those later, they are really nice for testing